### ISSUE TITLE
Chip manufacturers as separate stakeholder

### ISSUE THREAD

**Issue by Giszmo** [bitcoin-cap/bcap#34]
The "Chip manufacuters" in the list of miners comes a bit as a surprise and would warrant a sentence as to why it was included here. If not, it should be listed as a separate stakeholder. Bitmain's opposition to SegWit over their covert ASIC boost comes to mind.

**Comment by moneyball** [comment]
I like this idea to create a table with mining segments https://github.com/bitcoin-cap/bcap/issues/19

Chip makers would be one of them, and BCAP can do a better job of distinguishing chip makers from pools and miners (hashers).

**Comment by jaybeddict1** [comment]
agree, will work on #19  and include Chip Manufactures as a distinct component of the mining segment.

**Comment by Gudnessuche** [comment]
Agree.

When most people wake up to how much the Han Chinese control this sector, I hope it wouldn't be too late
